### PR TITLE
Add Cilium CNI

### PR DESCRIPTION
**Pull request by techcet**
Signed-off-by: Cynthia Thomas <cynthia@covalent.io>

### CODE REVIEW

File: docs/cni.md [hunk 1]
```diff
@@ -487,6 +488,18 @@ guide on adding Calico-CNI to
 Mesos](https://github.com/projectcalico/calico-containers/blob/master/docs/mesos/ManualInstallCalicoCNI.md).
```
Comment (jieyu) [inline docs/cni.md]: kill one line to be consistent with other places

File: docs/cni.md [hunk 2]
```diff
@@ -487,6 +488,18 @@ guide on adding Calico-CNI to
...
 
 
+#### <a name="a-cilium-network">A Cilium network</a>
```
Comment (jieyu) [inline docs/cni.md]: Please remove tailing white spaces. It failed our style checker:
```Jies-MacBook-Pro:mesos jie$ support/apply-review.sh -g 232
Cookie coming from patch-diff.githubusercontent.com attempted to set domain to github.com
2017-09-06 10:26:30 URL:https://patch-diff.githubusercontent.com/raw/apache/mesos/pull/232.patch [1509] -> "232.patch" [1]
232.patch:31: trailing whitespace.
Cilium brings HTTP-aware network security filtering to Linux container
232.patch:32: trailing whitespace.
frameworks. Using a new Linux kernel technology called BPF, Cilium
232.patch:33: trailing whitespace.
provides a simple and efficient way to define and enforce both
232.patch:36: trailing whitespace.
For more information on using Cilium with Mesos, check out the
warning: 4 lines add whitespace errors.
docs/cni.md:495: trailing whitespace.
+Cilium brings HTTP-aware network security filtering to Linux container 
docs/cni.md:496: trailing whitespace.
+frameworks. Using a new Linux kernel technology called BPF, Cilium 
docs/cni.md:497: trailing whitespace.
+provides a simple and efficient way to define and enforce both 
docs/cni.md:500: trailing whitespace.
+For more information on using Cilium with Mesos, check out the ```

**Comment by techcet** [comment]
@jieyu How does it look now?

**Comment by jieyu** [comment]
Merged!

**Pull request closed without merging**